pkg/chartverifier/certifier.go

```diff
@@ -69,10 +78,11 @@ func (c *certifier) Certify(uri string) (*Certificate, error) {
 
 	osVersion, err := oc.GetVersion()
 	if err != nil {
-		// NOTE(isutton): oc or a suitable kubeconfig might not be
-		//                available, so for now we leave it as "N/A"
-		//                instead of raising an error.
-		osVersion = "N/A"
+		if c.openshiftVersion != "" {
+			osVersion = c.openshiftVersion
+		} else {
+			return nil, OpenShiftVersionErr{err.Error()}
+		}
 	}
```

Comment:
Perhaps inverting the `if/else` clause would read better, as in wrapping the `return` in the `if` clause. Another option would be coalescing the `err` and the `c.openshiftVersion` check: `if err != nil && c.openshiftVersion == "" { return nil, ... } else { osVersion = c.openshiftVersion }`.

Reply:
Done. bd97de524900698c5badb59f30df389f1b827d63